R/pool.R

```diff
@@ -1,6 +1,6 @@
 
 
-#' Pool analysis results
+#' Pool analysis results across imputed datasets
```

Comment:
@nociale ,  I am happy with all the changes here. This one here was the only one I wasn't sure about as we aren't using the imputed datasets at the stages but the results from the imputed datasets ?  I am not sure if I am just being too pedantic though, what do you think ?

Reply:
Yes I understand what you mean. We actually don't use the imputed datasets here, but only the analysis results. I don't have a strong preference, maybe if we want to be extremely precise we could say "Pool analysis results obtained from the imputed datasets" or similar? @wolbersm What do you think?

Reply:
Thanks! I agree and like "Pool analysis results obtained from the imputed datasets". Have edited this branch accordingly.